tests/src/podman-machine-onboarding.spec.ts

```diff
@@ -0,0 +1,197 @@
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Page } from 'playwright';
+import type { RunnerTestContext } from './testContext/runner-test-context';
+import { afterAll, beforeAll, test, describe, beforeEach } from 'vitest';
+import { PodmanDesktopRunner } from './runner/podman-desktop-runner';
+import { WelcomePage } from './model/pages/welcome-page';
+import type { DashboardPage } from './model/pages/dashboard-page';
+import type { Locator } from '@playwright/test';
+import { expect as playExpect } from '@playwright/test';
+import { NavigationBar } from './model/workbench/navigation';
+import type { SettingsBar } from './model/pages/settings-bar';
+import { ResourcesPage } from './model/pages/resources-page';
+import { deletePodmanMachine } from './utility/operations';
+import { PodmanOnboardingPage } from './model/pages/podman-onboarding-page';
+
+const PODMAN_MACHINE_STARTUP_TIMEOUT: number = 360_000;
+
+let pdRunner: PodmanDesktopRunner;
+let page: Page;
+let dashboardPage: DashboardPage;
+let resourcesPage: ResourcesPage;
+let settingsBar: SettingsBar;
+let navigationBar: NavigationBar;
+let podmanOnboardingPage: PodmanOnboardingPage;
+
+let notificationPodmanSetup: Locator;
+let podmanMachineConfiguration: Locator;
+
+beforeAll(async () => {
+  pdRunner = new PodmanDesktopRunner();
+  page = await pdRunner.start();
+  pdRunner.setVideoAndTraceName('podman-machine-e2e');
+
+  const welcomePage = new WelcomePage(page);
+  await welcomePage.handleWelcomePage(true);
+  navigationBar = new NavigationBar(page);
+
+  // Delete machine if it already exists
+  await deletePodmanMachine(page, 'Podman Machine');
+});
+
+afterAll(async () => {
+  await pdRunner.close();
+}, 120000);
+
+beforeEach<RunnerTestContext>(async ctx => {
+  ctx.pdRunner = pdRunner;
+});
+
+describe('Podman Machine verification', async () => {
+  describe('Podman Machine onboarding workflow', async () => {
+    test('Setup Podman push notification is present', async () => {
+      dashboardPage = await navigationBar.openDashboard();
+      await playExpect(dashboardPage.mainPage).toBeVisible();
+      await playExpect(dashboardPage.notificationsBox).toBeVisible();
+      notificationPodmanSetup = dashboardPage.notificationsBox
+        .getByRole('region')
+        .filter({ hasText: 'Podman needs to be set up' });
+      await playExpect(notificationPodmanSetup).toBeVisible();
+    });
+    describe.skip('Onboarding navigation', async () => {
```

Comment:
Why is this describe set to be skipped?

Reply:
that was just temporary while we were trying something with Ondra